src/os/bluestore/BlueStore.cc

```diff
@@ -5966,6 +5985,8 @@ int BlueStore::_fsck(bool deep, bool repair)
   if (r < 0)
     goto out_fm;
 
+  _set_finisher_num();
```

Comment:
IMO it's a bit better to put this call along with other similar calls to _open_super_meta(). Just to have a single location for all the stuff of that sort... 
Just a thought, not pushing on that...